From bdd9587ac00d9b503144e72e82c9e88b08880f21 Mon Sep 17 00:00:00 2001
From: Julien Gilli <julien.gilli@joyent.com>
Date: Thu, 23 Nov 2017 16:27:32 -0800
Subject: [PATCH] PUBAPI-1460 nightly1-080-test-cloudapi fails in
 volumes-automount.test.js Reviewed by: Trent Mick <trentm@gmail.com> Approved
 by: Trent Mick <trentm@gmail.com>

---
 lib/post-setup/dev-sample-data.js | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/lib/post-setup/dev-sample-data.js b/lib/post-setup/dev-sample-data.js
index aeda52b..08149bf 100644
--- a/lib/post-setup/dev-sample-data.js
+++ b/lib/post-setup/dev-sample-data.js
@@ -78,7 +78,13 @@ function addDevSamplePkgs(opts, cb) {
 function addDevSampleImgs(opts, cb) {
     var imgNames = [
         'base-64-lts',
-        'minimal-64-lts'
+        /*
+         * minimal-64-lts, ubuntu-16.04 and ubuntu-certified-16.04 images are
+         * needed (among other things) by CloudAPI's volumes-automount.test.js.
+         */
+        'minimal-64-lts',
+        'ubuntu-16.04',
+        'ubuntu-certified-16.04'
     ];
     var imagesJo = opts.sdcadm.imagesJo;
     var imgapi = opts.sdcadm.imgapi;
-- 
2.21.0

